application/controllers/service_helper.py

```diff
@@ -4,85 +4,17 @@
 import structlog
 from flask import current_app
 
-from application.exceptions import RasError
+from application.exceptions import RasError, ServiceUnavailableException
 
 log = structlog.wrap_logger(logging.getLogger(__name__))
 
 
-def get_case_group(case_id):
```

Comment:
All unused so I got rid, we do need to circle back around as there are other things that can be removed